src/components/navigation/nav-tab-panel.tsx

```diff
@@ -50,7 +50,7 @@ export class NavTabPanel extends BaseComponent<IProps> {
                                 ? `calc(${dividerPosition}% - ${collapseTabWidth}px + 1px)`
                                 : `calc(${dividerPosition}% - ${resizePanelWidth}px - 4px)`;
     const resourceWidthStyle = {width: resourceWidth};
-    const isChatEnabled = true; // user.type === "teacher" if we want to restrict students
+    const isChatEnabled = user.type === "teacher"; //only enable chat for teachers
```

Comment:
Should this be `user.isNetworkedTeacher` like in the other two files?

Reply:
@tealefristoe Sure. I suppose I just followed an earlier commit and just changed it so that it would match a previous version of main - but this makes sense to follow a pattern along with the other files.